### ISSUE TITLE
Missing template for intra-SIG governance

### ISSUE THREAD

**Issue by daminisatya** [kubernetes/community#3544]
There is a short template file missing for intra-SIG governance in the below file. https://github.com/kubernetes/community/blob/master/governance.md

```
A short template for intra-SIG governance has been developed in order to simplify SIG creation, and additional templates are being developed, but SIGs should be relatively free to customize or change how they operate, within some broad guidelines and constraints imposed by cross-SIG processes (e.g., the release process) and assets (e.g., the kubernetes repo).
```

I think the link to the template is either deleted or missing. ```https://github.com/kubernetes/community/blob/master/committee-steering/governance/sig-governance-template-short.md```

If the template needs to be created, I'm happy to help. 

@nikhita @parispittman

**Comment by nikhita** [comment]
/sig contributor-experience
/committee steering

I guess this would fall under https://github.com/kubernetes/steering/issues/99?

cc @dims

**Comment by cblecker** [comment]
Fixed in #3728
